### PR TITLE
Match assembly version to package version, simplify csproj, and address Visual Studio IDE complaints

### PR DESCRIPTION
**Pull request by jnm2**
Hoping to be helpful, but feel free to reject any/all commits.

### CODE REVIEW

File: src/HidLibrary/HidLibrary.csproj [hunk 1]
```diff
@@ -1,74 +1,28 @@
...
-    <RootNamespace>HidLibrary</RootNamespace>
-    <AssemblyName>HidLibrary</AssemblyName>
     <TargetFrameworks>net20;net35;net40;net45;netstandard2</TargetFrameworks>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
 
+    <Version>3.2.49</Version>
+    <Company>Ultraviolet Catastrophe</Company>
+    <Copyright>Copyright © 2011 Ultraviolet Catastrophe</Copyright>
```
Comment (amullins83) [inline src/HidLibrary/HidLibrary.csproj:8]:
```suggestion
    <Copyright>Copyright © 2011-2020 Ultraviolet Catastrophe</Copyright>

```

File: src/TestHarness/Main.Designer.cs [hunk 2]
```diff
@@ -39,8 +39,8 @@ private void InitializeComponent()
             // 
             // Devices
             // 
-            this.Devices.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left)
-                        | System.Windows.Forms.AnchorStyles.Right)));
+            this.Devices.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
+            | System.Windows.Forms.AnchorStyles.Right)));
```
Comment (amullins83) [inline src/TestHarness/Main.Designer.cs:43]: We probably didn't have the right amount of whitespace here before, but I think it reads better with at least a space or two to indicate that it's a continuation of the above line. It also seems odd to me that we separate the Top, Left, and Right flags with parentheses like that (i.e. it looks like this was odd before, and I just never noticed until now).

File: src/TestHarness/Main.Designer.cs [hunk 3]
```diff
@@ -102,9 +102,9 @@ private void InitializeComponent()
             // 
             // Output
             // 
-            this.Output.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom)
-                        | System.Windows.Forms.AnchorStyles.Left)
-                        | System.Windows.Forms.AnchorStyles.Right)));
+            this.Output.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
+            | System.Windows.Forms.AnchorStyles.Left) 
+            | System.Windows.Forms.AnchorStyles.Right)));
```
Comment (amullins83) [inline src/TestHarness/Main.Designer.cs:107]: Ignore me if this is auto-generated, but the parentheses separating the different flags seems really weird to me, as well as continuing lines without any indentation.

Comment (jnm2) [reply src/TestHarness/Main.Designer.cs:107]: This is autogenerated. I didn't even look at this file to be honest :D

File: src/TestHarness/Main.Designer.cs (hunk 2 above)
Comment (jnm2) [reply src/TestHarness/Main.Designer.cs:43]: (This file is autogenerated as mentioned in the comment below)

**Review by amullins83 — APPROVED** [review]

**Pull request merged**